actions/src/main/java/com/redhat/service/bridge/actions/ActionParameterValidator.java

```diff
@@ -0,0 +1,7 @@
+package com.redhat.service.bridge.actions;
+
+import com.redhat.service.bridge.infra.models.actions.BaseAction;
+
+public interface ActionParameterValidator {
+    boolean isValid(BaseAction baseAction);
```

Comment:
I would return a collection of violation instead of a boolean so that we can provide meaningful error reporting.
Wdyt?
```suggestion
    Set<ValidationProblem> validate(BaseAction baseAction);
```

Reply:
I was thinking to return a `ValidationResult` with `isValid()` and a `getMessage()` method. The message contains the validation error string and can be constructed in any way by the `ActionParameterValidator`. So final code here would look something like:

```
ValidationResult validationResult = actionProvider.getParameterValidator().isValid(baseAction);
if(!validationResult.isValid()) {
   context.buildConstraintViolationWithTemplate(validationResult.getMessage()).addConstraintViolation();
}
return validationResult.isValid();
```

Reply:
Fine for me 👍